<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyDialog</class>
 <widget class="QDialog" name="KeyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget_keyboard" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>609</width>
     <height>533</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton_keyboard_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>126</width>
      <height>98</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_keyboard_2">
    <property name="geometry">
     <rect>
      <x>171</x>
      <y>90</y>
      <width>126</width>
      <height>98</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_keyboard_3">
    <property name="geometry">
     <rect>
      <x>312</x>
      <y>90</y>
      <width>126</width>
      <height>98</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_keyboard_4">
    <property name="geometry">
     <rect>
      <x>453</x>
      <y>90</y>
      <width>126</width>
      <height>98</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_keyboard_8">
    <property name="geometry">
     <rect>
      <x>453</x>
      <y>198</y>
      <width>126</width>
      <height>98</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_keyboard_7">
    <property name="geometry">
     <rect>
      <x>312</x>
      <y>198</y>
      <width>126</width>
      <height>98</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_keyboard_6">
    <property name="geometry">
     <rect>
      <x>171</x>
      <y>198</y>
      <width>126</width>
      <height>98</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_keyboard_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>198</y>
      <width>126</width>
      <height>98</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_keyboard_resetting">
    <property name="geometry">
     <rect>
      <x>453</x>
      <y>306</y>
      <width>126</width>
      <height>98</height>
     </rect>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_keyboard_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>306</y>
      <width>126</width>
      <height>98</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_keyboard_x">
    <property name="geometry">
     <rect>
      <x>312</x>
      <y>306</y>
      <width>126</width>
      <height>98</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_keyboard_10">
    <property name="geometry">
     <rect>
      <x>171</x>
      <y>306</y>
      <width>126</width>
      <height>98</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_keyboard_back">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>425</y>
      <width>268</width>
      <height>78</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_keyboard_ok">
    <property name="geometry">
     <rect>
      <x>311</x>
      <y>425</y>
      <width>268</width>
      <height>78</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_keyboard">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>549</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
